allure-report-face/src/styles.css

```diff
@@ -66,6 +66,11 @@ html, body {
   white-space: nowrap;
 }
 
+.description__text {
```

Comment:
this class should be named `preformatted-text`.
We are strictly following BEM naming convention, so this file is supposed to contain only class names with general functionality.